{'description': {'en': 'User overlay'},
 'feed': ['https://cgit.gentoo.org/user/palmer.git/atom/'],
 'homepage': 'https://cgit.gentoo.org/user/palmer.git/',
 'name': 'palmer',
 'owner': [{'email': 'palmer@dabbelt.com',
            'name': 'Palmer Dabbelt',
            'type': 'person'}],
 'quality': 'experimental',
 'source': [{'type': 'git',
             'uri': 'git://anongit.gentoo.org/user/palmer.git'},
            {'type': 'git',
             'uri': 'https://anongit.gentoo.org/git/user/palmer.git'},
            {'type': 'git',
             'uri': 'git+ssh://git@git.gentoo.org/user/palmer.git'}],
 'status': 'unofficial'}
$ pmaint sync palmer
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
From git://anongit.gentoo.org/user/palmer
   97e15e9..c5c956b  master     -> origin/master
Updating 97e15e9..c5c956b
Fast-forward
 net-misc/livestreamer/Manifest                   |  2 ++
 net-misc/livestreamer/livestreamer-1.12.2.ebuild | 24 ++++++++++++++++++++++++
 2 files changed, 26 insertions(+)
 create mode 100644 net-misc/livestreamer/livestreamer-1.12.2.ebuild
*** syncing palmer
*** synced palmer
 * Sync succeeded
$ pmaint regen -t 2 palmer
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Error running an operation:
 Unhandled exception in regen_cache
Exception pkgcore.ebuild.processor.TimeoutError: TimeoutError() in <functools.partial object at 0x7f94764234c8> ignored
Traceback (most recent call last):
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 91, in shutdown_all_processors
    ignore_keyboard_interrupt=True)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 670, in shutdown_processor
    elif self.is_alive:
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 649, in is_alive
    if not self.expect("yep!", timeout=0.1):
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 500, in expect
    return self._consume_async_expects()
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 465, in _consume_async_expects
    got = [x.rstrip('\n') for x in self.readlines(len(self._outstanding_expects))]
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 505, in readlines
    mydata.append(self.ebd_read.readline())
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 472, in _timeout_ebp
    raise TimeoutError()
TimeoutError
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/spawn.py", line 89, in run_exitfuncs
    func(*targs, **kargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 70, in _inner
    return functor(*args, **kwds)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 91, in shutdown_all_processors
    ignore_keyboard_interrupt=True)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 670, in shutdown_processor
    elif self.is_alive:
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 649, in is_alive
    if not self.expect("yep!", timeout=0.1):
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 500, in expect
    return self._consume_async_expects()
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 465, in _consume_async_expects
    got = [x.rstrip('\n') for x in self.readlines(len(self._outstanding_expects))]
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 505, in readlines
    mydata.append(self.ebd_read.readline())
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 472, in _timeout_ebp
    raise TimeoutError()
TimeoutError
Error in sys.exitfunc:
ERROR pkgcore: 
Traceback (most recent call last):
  File "/usr/lib64/python2.7/atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/spawn.py", line 89, in run_exitfuncs
    func(*targs, **kargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 70, in _inner
    return functor(*args, **kwds)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 91, in shutdown_all_processors
    ignore_keyboard_interrupt=True)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 670, in shutdown_processor
    elif self.is_alive:
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 649, in is_alive
    if not self.expect("yep!", timeout=0.1):
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 500, in expect
    return self._consume_async_expects()
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 465, in _consume_async_expects
    got = [x.rstrip('\n') for x in self.readlines(len(self._outstanding_expects))]
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 505, in readlines
    mydata.append(self.ebd_read.readline())
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 472, in _timeout_ebp
    raise TimeoutError()
pkgcore.ebuild.processor.TimeoutError
 * Cache regen failed with 246
